.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,67 @@
+name: npm-publish
+
+on:
+  workflow_call:
+    inputs:
+      dir:
+        type: string
+        required: true
+    secrets:
+      NPM_TOKEN:
+        required: true
+
+env:
+  SCOPE: '@taqueria'
+  NPM_REGISTRY: 'https://registry.npmjs.org/'
+  GH_REGISTRY: 'https://npm.pkg.github.com/'
+
+jobs:
+  build-and-publish-npm-packages:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+      contents: read
+    defaults:
+      run:
+        working-directory: ${{ inputs.dir }}
+    env: 
+      NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with: 
+          registry-url: ${{ env.NPM_REGISTRY }}
+          scope: "${{ env.SCOPE }}"
+
+      - name: Check tag suffix
+        id: check-tag-suffix
+        if: startsWith(github.ref, 'refs/tags/v') && (contains(github.ref, '-rc') || contains(github.ref, '-beta') || contains(github.ref, '-test'))
+        run: |
+          echo "::set-output name=npm-tag::next"
+
+      - name: Prepare package.json for publishing
+        run: npm run prod-config-ci
```

Comment:
Same as for the `build-ci` script; why can't we use the normal `prod-config` script?